stdlib/list.mli

```diff
@@ -359,6 +359,44 @@ val filteri : (int -> 'a -> bool) -> 'a list -> 'a list
    @since 4.11
 *)
 
+
+(** {1 List manipulation} *)
+
+
+val take : int -> 'a list -> 'a list
+(** [take n l] returns the prefix of [l] of length [n],
+    or a copy of [l] if [n > length l].
```

Comment:
`or a copy of [l] if [n >= length l]`
(similarly for drop)

Reply:
done